### PR TITLE
fix/not found assets dir and incorrectly loaded stylesheet

### PR DESCRIPTION
**Pull request by emil-placheta**

### CODE REVIEW

File: web/src/util/build.js [hunk 1]
```diff
@@ -8,9 +8,11 @@ import config from '../app/config.js';
 const staticDir = config.staticDir;
 const require = createRequire(import.meta.url);
 // resolves to the node_modules folder with govuk-frontend and @ministryofjustice/frontend in
-const govUkNodeModules = path.resolve(require.resolve('govuk-frontend'), '../../../..');
+const govUkFrontend = path.resolve(require.resolve('govuk-frontend'), '../../../..');
 const mojNodeModules = path.resolve(require.resolve('@ministryofjustice/frontend'), '../../../..');
 
+console.log('CONFIG::>>>>', config);
```
Comment (beejjacobs) [inline web/src/util/build.js:14]: let's remove this, I assume it was for debugging.

File: web/src/util/build.js [hunk 2]
```diff
@@ -8,9 +8,11 @@ import config from '../app/config.js';
 const staticDir = config.staticDir;
 const require = createRequire(import.meta.url);
 // resolves to the node_modules folder with govuk-frontend and @ministryofjustice/frontend in
-const govUkNodeModules = path.resolve(require.resolve('govuk-frontend'), '../../../..');
+const govUkFrontend = path.resolve(require.resolve('govuk-frontend'), '../../../..');
 const mojNodeModules = path.resolve(require.resolve('@ministryofjustice/frontend'), '../../../..');
```
Comment (beejjacobs) [inline web/src/util/build.js:12]: The comment is out of date, and let's use the same suffix for govUk and moj how about:

- `govUkRoot` & `mojRoot` ?
- comment: 

```
// resolves to <root>/node_modules/govuk-frontend/govuk/all.js than maps to `<root>`
const govukRoot ...
// resolves to <root>/node_modules/@ministryofjustice/frontend/moj/all.js then maps to `<root>`
const mojRoot ...
```

**Review by beejjacobs — COMMENTED** [review]
Couple of points :)

File: web/src/util/build.js (hunk 1 above)
Comment (emil-placheta) [reply web/src/util/build.js]: oops

File: web/src/util/build.js (hunk 2 above)
Comment (emil-placheta) [reply web/src/util/build.js]: updated

**Review by beejjacobs — APPROVED** [review]

**Pull request merged**